### PR TITLE
Radxa rock5 b upstream

### PR DESCRIPTION
**Pull request by nascs**
Hi @CurlyMoo, can you please take a look at the pr? I've finished testing it.

### CODE REVIEW

File: src/platform/radxa/common.c [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/*
+	Copyright (c) 2023 Radxa Ltd.
+	Author: Nascs <nascs@radxa.com>
+					ZHANG Yuntian <yt@radxa.com>
```
Comment (CurlyMoo) [inline src/platform/radxa/common.c]: Indentation is off

File: src/soc/rockchip/rk3588.c [hunk 2]
```diff
@@ -0,0 +1,421 @@
...
+	return rockchipDigitalRead(rk3588, i);
+}
+
+static int rk3588PinMode(int i, enum pinmode_t mode) {
+	struct layout_t *pin = NULL;
+	unsigned int *cru_reg = NULL;
+	unsigned int *grf_reg = NULL;
+	unsigned int *dir_reg = NULL;
+
+	if( (pin = rockchipGetPinLayout(rk3588, i)) == NULL) {
```
Comment (CurlyMoo) [inline src/soc/rockchip/rk3588.c]: Style difference

File: src/soc/rockchip/rk3588.c [hunk 3]
```diff
@@ -0,0 +1,421 @@
...
+}
+
+static int rk3588WaitForInterrupt(int i, int ms) {
+	return rockchipWaitForInterrupt(rk3588, i, ms);
+}
+
+static int rk3588GC(void) {
+	rockchipGC(rk3588);
+
+	if (cru_ns_register_virtual_address != NULL) {
```
Comment (CurlyMoo) [inline src/soc/rockchip/rk3588.c]: Style guide is inconsistent here

File: src/platform/radxa/common.c (hunk 1 above)
Comment (nascs) [reply src/platform/radxa/common.c]: Is it because I'm using a tab size equal to 2 spaces?

Comment (nascs) [reply src/platform/radxa/common.c]: Hi @CurlyMoo, can you look at this pr again? I've  made the changes.

Comment (nascs) [reply src/platform/radxa/common.c]: Hi @CurlyMoo, I would appreciate it if you have time to look at this pr again.

Comment (nascs) [reply src/platform/radxa/common.c]: Hi @CurlyMoo, I hope everything is going well. I really appreciate it that if you could take the time to help review this pr again.

**Comment by RadxaYuntian** [comment]
Hi @CurlyMoo . Just checking if you are still around to give this PR another review.

**Comment by CurlyMoo** [comment]
@RadxaYuntian I do everytime you ask but the still pending issues are not being resolved.

**Comment by CurlyMoo** [comment]
There are still pending requests open.

**Comment by RadxaYuntian** [comment]
Just to clarify do you mean we need to "resolve" the review comments by clicking the resolve conversation button, or do you mean there are unfixed issues still present in the code? In our own review process we let the reviewer to click the resolve conversation button, so they can track what is actually fixed and what is still remaining.

**Comment by peterwang2050** [comment]
Hi @CurlyMoo is there anything blocked?  can you reviewer it again.

**Pull request merged**